raycicmd/config.go

```diff
@@ -165,13 +165,16 @@ var branchPipelineConfig = &config{
 	},
 
 	RunnerQueues: map[string]string{
-		"default":   "runner_queue_small_branch",
-		"small":     "runner_queue_small_branch",
-		"medium":    "runner_queue_medium_branch",
-		"large":     "runner_queue_branch",
-		"gpu":       "gpu_runner_queue_branch",
-		"gpu-large": "gpu_large_runner_queue_branch",
-		"trainium":  "trainium_runner_queue_branch",
+		"default":     "runner_queue_small_branch",
+		"small":       "runner_queue_small_branch",
+		"medium":      "runner_queue_medium_branch",
+		"large":       "runner_queue_branch",
+		"gpu":         "gpu_runner_queue_branch",
+		"gpu-large":   "gpu_large_runner_queue_branch",
+		"trainium":    "trainium_runner_queue_branch",
+		"window":      "windows_queue_branch",
```

Comment:
could you change `window` to `win` or `windows`?

Reply:
and at this point I think we should add step skipping based on jobenv / instance types now..